### PR TITLE
Fix creating policies from template

### PR DESCRIPTION
**Pull request by deiferni**
There was an error with cyclic imports during policy creation. This PR fixes that issue by moving a class to its own module from which it can be imported separately.

It also changes the `create_client` script to always pass remaining arguments to the template engine (mr.bob).

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross plz have a look

File: opengever/base/pathfinder.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from App.config import DefaultConfiguration
+import App.config
+import os
+
+
+class PathFinder(object):
+    """Helper class to provide various Zope2 Instance related paths that are
+    otherwise cumbersone to access.
```
Comment (phgross) [inline opengever/base/pathfinder.py]: typo ...

**Comment by phgross** [comment]
:+1:

**Comment by deiferni** [comment]
@phgross updated

**Pull request merged**